Framework/SwiftOSC/Communication/OSCServer.swift

```diff
@@ -11,36 +16,70 @@ extension OSCServerDelegate {
     public func didReceive(_ message: OSCMessage){}
 }
 
+
+
+
+
+/// Used to receive OSC messages from another source, which are sent to the delegate.
 open class OSCServer {
+    
+    /// Returns the current address, or sets the address messages will be sent to.
     open var address: String {
         didSet {
             _ = server.close()
             server = UDPServer(addr: self.address, port:self.port)
         }
     }
+    
+    /// Returns the current port, or sets the port messages will be sent to.
     open var port: Int {
         didSet {
             _ = server.close()
             server = UDPServer(addr: self.address, port:self.port)
         }
     }
+    
+    /// Callbacks are send to this instance.
     open var delegate: OSCServerDelegate?
-    open var running = false
+    
+    /// Returns the state of the server if its enabled or not.
+    open fileprivate(set) var running = false
+    
     var server: UDPServer
     
+    
+    /**
+     Creates a new OSCServer
+     
+     - Parameters:
+       - address: Could be an IP address, or just "localhost" for sending internally in your computer. For most cases, just leave this with a blank String
```

Comment:
Let's change this. 

- address: The address of the network interface for receiving data. Leave blank to receive from all locations. Leave blank unless you really understand what you are doing.

Reply:
👍